### PR TITLE
feat: chat context and hook

### PR DESCRIPTION
**Pull request by LomyW**
- [x] context to create session
- [x] added useChat hook

### CODE REVIEW

File: src/context/ChatContext.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    const me = {
+        id: '123',
+        name: 'Lomy',
+        role: 'student',
+        email: 'salome.wick@typedigital.de',
+    };
+
+    useEffect(() => {
+        Talk.ready.then(() => markTalkLoaded(true));
```
Comment (Jonasdoubleyou) [inline src/context/ChatContext.tsx:31]: Registered multiple times, should be in a separate useEffect with no dependencies

File: src/context/ChatContext.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                appId: 't5NarFaG',
+                me: currentUser,
+                // signature:
+            });
+            setSession(session);
+            return () => session.destroy();
+        }
+    }, [talkLoaded]);
+
+    return <ChatContext.Provider value={{ session, talkLoaded }}>{children}</ChatContext.Provider>;
```
Comment (Jonasdoubleyou) [inline src/context/ChatContext.tsx:46]: I think we can prevent quite some rerenders by moving the context value into a useMemo

File: src/components/WithNavigation.tsx [hunk 3]
```diff
@@ -58,6 +50,16 @@ const WithNavigation: React.FC<Props> = ({
         lg: space['1'],
     });
 
+    const { t } = useTranslation();
+
+    const navItems: NavigationItems = {
+        start: { label: t('navigation.label.start'), icon: LFHomeIcon },
+        appointments: { label: t('navigation.label.appointments'), icon: LFAppointmentIcon },
+        chat: { label: t('navigation.label.chat'), icon: LFChatIcon },
```
Comment (Jonasdoubleyou) [inline src/components/WithNavigation.tsx:58]: if you want to bring this to main, please remove not yet finished pages from the nav bar

Comment (LomyW) [reply src/components/WithNavigation.tsx]: switched the base branch 😄

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**